mattermost/desktop
# [MM-25429]/[MM-25433] fix finder focus, align buttons, keep state consistent

Before submitting, please confirm you've

Please provide the following information:

**Summary**

Focus was being lost, due to click went through and was also captured by the tabbar, which then autofocused itself. Since the searchbar was probably misplaced, I moved it to where it seemed to look right. Also fixed misalignment of buttons mentioned on the ticket.

**Issue link**

[MM-25429](https://mattermost.atlassian.net/browse/MM-25429)
[MM-25433](https://mattermost.atlassian.net/browse/MM-25433)

**Test Cases**

Steps:

1. open searchbar (cmd+f)
1. click outside
1. click on searchbar

Result:

Focus should remain on searchbar

**Additional Notes**

@andrewbrown00 should we create a separate story for styling the search bar in dark mode?

**Screenshots**
mac
<img width="1911" alt="Screenshot 2020-05-27 at 11 21 44" src="https://user-images.githubusercontent.com/1515906/83002467-6358bb80-a00d-11ea-9e01-fb121354314d.png">
win
<img width="541" alt="Screenshot 2020-05-27 at 11 35 34" src="https://user-images.githubusercontent.com/1515906/83003153-35c04200-a00e-11ea-816e-9b69f7b5e1d9.png">
